test/CBLTest.cc

```diff
@@ -128,10 +128,10 @@ cbl::Database CBLTest_Cpp::openDatabaseNamed(fleece::slice name) {
 void CBLTest_Cpp::createNumberedDocs(cbl::Collection& collection, unsigned n, unsigned start) {
     for (unsigned i = 0; i < n; i++) {
         char docID[20];
-        sprintf(docID, "doc-%03u", start + i);
+        snprintf(docID, 20, "doc-%03u", start + i);
```

Comment:
It would be a whole lot better to not use a magic number twice in all these cases

Reply:
Yes, let me do it.